client/widgetdecorations.cpp

```diff
@@ -170,6 +170,32 @@ void close_widget::notify_parent()
   fcw->update_menu();
 }
 
+/**
+   Add the resizable_type to the list (multiple types support)
+ */
+void resizable_widget::setResizable(QFlags<resizable_flag> flags)
+{
+  resizeFlags = flags;
+}
+
+QFlags<resizable_flag> resizable_widget::getResizable() const
+{
+  return resizeFlags;
+}
+
+/**
+   Remove the active resizable_type from list
+ */
+void resizable_widget::removeResizable() { resizeFlags = {}; }
+
+/**
+   Check the active resizable_type or not
+ */
+bool resizable_widget::hasResizable(resizable_flag flag) const
+{
+  return resizeFlags.testFlag(flag);
```

Comment:
> I guess it could be made inline

Are you talking about the inline specifier?
```cpp
inline bool resizable_widget::hasResizable(resizable_flag flag) const {}
```

Reply:
The specifier is mostly ignored by compilers nowadays, but we could help them by moving the implementation to the header. Anyway not a critical issue :)

Reply:
> The specifier is mostly ignored by compilers nowadays, but we could help them by moving the implementation to the header. Anyway not a critical issue :)

Yeah